pkgs/development/python-modules/qasm2image/default.nix

```diff
@@ -12,13 +12,13 @@
 
 buildPythonPackage rec {
   pname = "qasm2image";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
-    owner = "nelimeee";
+    owner = "pandaman64";
```

Comment:
What's the reason for switching to a fork? It seems to be behind the original repo.